tests/integration_tests/clouds.py

```diff
@@ -201,13 +207,28 @@ def delete_snapshot(self):
                 )
                 self.cloud_instance.delete_image(self.snapshot_id)
 
+    def _ignore_image_type(self, **kwargs) -> dict:
```

Comment:
Is this method actually required?

I see that `_get_initial_image()` is overriden in Ec2Cloud, GceCloud and AzureCloud, where they pass image_type, but since image_type doesn't appear to be passed in OciCloud, _LxdIntegreationCloud or OpenstackCloud I wonder whether we can drop this method and its callsites. I could see this being required if the base class's `_get_initial_image()` defaulted to pass `image_type=self._image_type` to `daily_image()`, but that's not the case so I'm not sure why this is required.

Please let me know your thoughts.

Reply:
You're right. As `_get_inital_image` is a protected method, we control who calls it, therefore there is no risk of a caller passing `image_type` for clouds not supporting it.

Removed, thanks.